src/sider/arbeidsuforhet/avslag/AvslagForm.tsx

```diff
@@ -58,15 +59,15 @@ const texts = {
 const begrunnelseMaxLength = 5000;
 
 export interface Props {
-  varselSvarfrist: Date;
+  sisteVurdering: VurderingResponseDTO;
 }
 
 export interface ArbeidsuforhetAvslagSkjemaValues {
   begrunnelse: string;
   fom: Date;
 }
 
-export function AvslagForm({ varselSvarfrist }: Props) {
+export function AvslagForm({ sisteVurdering }: Props) {
```

Comment:
Endret litt på signaturen her for å få med `created_at`-datapunktet på forhåndsvarselet.